### PR TITLE
fix: update light-black for inactive tabs

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/styles/theme/typography.scss [hunk 1]
```diff
@@ -74,7 +74,7 @@
     color: $color-grey-5;
 }
 .typography-color-light-black {
-    color: $color-black-6;
+    color: #000000b8;
```
Comment (hasan-deriv) [inline src/features/styles/theme/typography.scss:77]: Why have you changed the color value here? This class is using in several places. Those will be affected also.

Comment (Nuzhy-Deriv) [reply src/features/styles/theme/typography.scss:77]: this class was added by @mayuran. This class is only used for inactive tab menus and need to change in both tabs in option page

**Review by mayuran-deriv — APPROVED** [review]

File: src/features/styles/theme/typography.scss (hunk 1 above)
Comment (mayuran-deriv) [reply src/features/styles/theme/typography.scss:77]: ya its only used in my component

Comment (hasan-deriv) [reply src/features/styles/theme/typography.scss:77]: Then, it's ok

**Review by mitra-deriv — APPROVED** [review]

**Pull request merged**